example/lib/presentation/samples/gauge/gauge_chart_sample1.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:fl_chart_app/presentation/resources/app_resources.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:flutter/material.dart';
+
+class GaugeChartSample1 extends StatefulWidget {
+  const GaugeChartSample1({super.key});
+
+  @override
+  State<StatefulWidget> createState() => GaugeChartSample1State();
+}
+
+class GaugeChartSample1State extends State {
+  double _value = 0.7;
+  bool _isSelected = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(28),
+      child: Column(
+        children: [
+          SizedBox(
+            width: 250,
+            height: 250,
+            child: GaugeChart(
+              GaugeChartData(
+                value: _value,
```

Comment:
I think we can have a list of values (each value can have startAngle and endAngle). 
Then we can draw them on top of each other. 
This way, it is more customizable.

Look at these samples:

<img width="395" alt="image" src="https://github.com/imaNNeo/fl_chart/assets/7009300/8b9948d4-f2a8-4519-ac49-f2e6e3a2ea01">

<img width="315" alt="image" src="https://github.com/imaNNeo/fl_chart/assets/7009300/07c54d00-d018-4042-bc96-4ab31c11b84c">

<img width="201" alt="image" src="https://github.com/imaNNeo/fl_chart/assets/7009300/44afd64b-d32c-4b8c-b922-5360f99513bb">